engine-src/Game/LambdaHack/Server/Commandline.hs

```diff
@@ -184,6 +186,12 @@ newGameP = optional $ max 1 <$> min difficultyBound <$>
               <> help "Start a new game, overwriting the save file, with difficulty for all UI players set to N"
               <> metavar "N" )
 
+fullscreenP :: Parser Bool
+fullscreenP =
+  switch (  long "fullscreen"
+         <> short 'f'
+         <> help "Start game in fullscreen mode (currently, only implemented for the SDL renderer)" )
```

Comment:
We warn in README that only SDL and Web is maintained (if it's not clear, needs to clarified), so I'd simplify and remove the disclaimer about SDL. And in in the browser, while the power-users can change the config, it's not supported, and also I don't think they'd expect fullscreen or scaling to work, when they can do that via browser fullscreen or zoom. BTW, we'd need to CPP away, for the browser build, the fullscreen toggle in Settings menu, if we implement it.